ci/do_ci.sh

```diff
@@ -280,7 +280,7 @@ elif [[ "$CI_TARGET" == "bazel.compile_time_options" ]]; then
     "--define" "boringssl=fips"
     "--define" "log_debug_assert_in_release=enabled"
     "--define" "quiche=enabled"
-    "--define" "wasm=wavm"
+    "--define" "wasm=wasmtime"
```

Comment:
We need to either enable both runtimes here (if the LLVM conflict is resolved), or move Wasmtime to another build target, since we need to have coverage for all runtimes in the CI.

Reply:
Given the discussion above, I don't think resolving symbol conflicts in this PR (only by leveraging a small patch) is feasible. So I will add another build for Wasmtime. After the merge, I'll file an issue for symbol collision.

Reply:
@lizan Is there any existing build target I could set `--define wasm=wastime`, or should I add a new one for testing wasmtime?

Reply:
I think you can add a line below to do more test with wasmtime in same CI target. before we figure out #12574

Reply:
https://github.com/envoyproxy/envoy/pull/13932/commits/ec4d2ddd33fefa7cee8508aaba7d499b968b0b85 done